base/ltoutput.dtx

```diff
@@ -2592,12 +2653,62 @@
 \NewSocketPlug {build/column/outputbox}{footnotes-floats-legacy} {%
 %    \end{macrocode}
 %    In the legacy case we don't really want to take out the bottom
-%    skip, but rather then altering \cs{@outputbox@removebskip} in
+%    skip, but rather than altering \cs{@outputbox@removebskip} in
 %    \cs{makecol} to do nothing, which we would then have to undo in
 %    each other layout, we simply reinsert the dropped skip
 %    immediately again.
+%
+%    In the legacy case there is, however, one other thing that needs
+%    doing: if there was a \cs{vfil} at the bottom of the page (e.g.,
+%    from \cs{newpage}) and there are no footnotes, then we need to back up
+%    by the depth of the last line in the page box. Otherwise you
+%    might end up with incorrect alignment if there is a higher-order
+%    fill on the page (e.g., \cs{vfill}) that pushes the text material
+%    to the bottom because then the depth will not be taken into
+%    account by \TeX{}.
+%    
+%    Given, that a bottom skip was taken out earlier, we can look at
+%    the \cs{@outputbox@depth} value for this and back up by that
```

Comment:
What is "this"? 
And why can you "look at . . . for this"?? 
Maybe? : 
we find "this?" in . . .

Reply:
changed